server/tests/trip.test.js

```diff
@@ -200,6 +201,17 @@ describe('Trip test suite', () => {
         done(err);
       });
   });
+  it('13.should create a multi-city trip', (done) => {
```

Comment:
Remove the number and update the **notMangerToken**

Reply:
done